samples/nrf9160/http_update/application_update/README.rst

```diff
@@ -8,17 +8,19 @@ nRF9160: HTTP application update
    :depth: 2
 
 The HTTP application update sample demonstrates how to do a basic firmware over-the-air (FOTA) update.
-It uses the :ref:`lib_fota_download` library to download a file from a remote server and write it to flash.
+It uses the :ref:`lib_fota_download` library to download two image files from a remote server and program them to flash memory.
 
 Overview
 ********
 
-The sample connects to an HTTP server to download a signed firmware image.
-The image is generated when building the sample, but you must upload it to a server and configure where it can be downloaded.
-See `Specifying the image file`_ for more information.
+The sample connects to an HTTP server to download two signed firmware images:
+* One image has the ``application version`` set to ``1``.
+* The other has the ``application version`` set to ``2``.
 
-By default, the image is saved to the `MCUboot`_ bootloader secondary slot.
-To be used by MCUboot, the downloaded image must be signed using imgtool.
+The sample updates between the two images, either from *version 1* to *version 2* or from *version 2* to *version 1*.
+
+By default, the images are saved to the `MCUboot`_ second-stage bootloader secondary slot.
```

Comment:
@jonathannilsen Is this also still correct?

Reply:
Looks correct